course_access_groups/serializers.py

```diff
@@ -1,13 +1,75 @@
-from rest_framework import serializers
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, unicode_literals
 
+from rest_framework import serializers
+from openedx.core.lib.api.serializers import CourseKeyField
 from course_access_groups.models import (
-    CourseAccessGroup,
+    Group,
+    CourseGroup,
+    Membership,
+    Rule,
 )
 
 
 class GroupSerializer(serializers.ModelSerializer):
+    id = serializers.ReadOnlyField()
+    organization_name = serializers.ReadOnlyField(source='organization.name')
+
+    class Meta:
+        model = Group
+        fields = [
+            'id', 'name', 'description', 'organization', 'organization_name',
+        ]
+
+
+class MembershipSerializer(serializers.ModelSerializer):
+    id = serializers.ReadOnlyField()
```

Comment:
I have to think about using `ReadOnlyField` versus using the type identify field class like `DateTimeField` or `CharField` because these fields help provide context on the expected data type.

Reply:
I'm not sure what do you mean, because all of the APIs have create/list/update/delete methods. So it's not really read-only.

Reply:
we can live without it